src/components/Spawn/spawnHostModal/getFormSchema.tsx

```diff
@@ -502,26 +501,19 @@ export const getFormSchema = ({
           },
         },
       }),
-      ...(shouldRenderVolumeSelection && {
-        homeVolumeDetails: {
-          selectExistingVolume: {
-            "ui:widget": isVirtualWorkstation
-              ? widgets.RadioBoxWidget
-              : "hidden",
-          },
-          volumeSelect: {
-            "ui:widget": isVirtualWorkstation ? AntdSelect : "hidden",
-            "ui:allowDeselect": false,
-            "ui:data-cy": "volume-select",
-            "ui:disabledEnums": (volumes || [])
-              .filter((v) => !!v.hostID)
-              .map((v) => v.id),
```

Comment:
Remove this line because we already filter out the volumes that have a host ID in L366 so there's no need to disable them here.